common/viewer.cpp

```diff
@@ -1701,67 +1701,67 @@ namespace rs2
 
             if (streams[stream].is_stream_visible())
                 switch (stream_type)
-            {
                 {
-                    case RS2_STREAM_GYRO: /* Fall Through */
-                    case RS2_STREAM_ACCEL:
                     {
-                        auto motion = streams[stream].texture->get_last_frame().as<motion_frame>();
-                        if (motion.get())
-                        {
-                            auto axis = motion.get_motion_data();
-                            stream_mv.show_stream_imu(font1, stream_rect, axis, mouse);
-                        }
-                        break;
+                case RS2_STREAM_GYRO: /* Fall Through */
+                case RS2_STREAM_ACCEL:
+                {
+                    auto motion = streams[stream].texture->get_last_frame().as<motion_frame>();
+                    if (motion.get())
+                    {
+                        auto axis = motion.get_motion_data();
+                        stream_mv.show_stream_imu(font1, stream_rect, axis, mouse);
                     }
-                    case RS2_STREAM_POSE:
+                    break;
+                }
+                case RS2_STREAM_POSE:
+                {
+                    if (streams[stream].show_stream_details)
                     {
-                        if (streams[stream].show_stream_details)
+                        auto pose = streams[stream].texture->get_last_frame().as<pose_frame>();
+                        if (pose.get())
                         {
-                            auto pose = streams[stream].texture->get_last_frame().as<pose_frame>();
-                            if (pose.get())
-                            {
-                                auto pose_data = pose.get_pose_data();
-                                if (stream_rect.contains(win.get_mouse().cursor))
-                                    stream_mv.show_stream_pose(font1, 
-                                    stream_rect, pose_data, 
+                            auto pose_data = pose.get_pose_data();
+                            if (stream_rect.contains(win.get_mouse().cursor))
+                                stream_mv.show_stream_pose(font1,
+                                    stream_rect, pose_data,
                                     stream_type, (*this).fullscreen, 30.0f, *this);
-                            }
                         }
-  
-                        break;
+                    }
+
+                    break;
+                }
```

Comment:
Indentation doesn't look right

Reply:
indentation was added after a document align command , I reverted the alignment